_episodes_rmd/20-r-rstudio.Rmd

```diff
@@ -21,8 +21,8 @@ source("../bin/chunk-options.R")
 ## What is R? What is RStudio?
 
 The term [R](https://www.r-project.org/) is used to refer to both the
-*programming language*, the *environment for statistical computing*
-and *the software* that interprets the scripts written using it.
+*programming language* and the *environment for statistical computing*
+i.e. *the software* that interprets the scripts written using it.
```

Comment:
I'm not convinced by this change. I think there is a distinction to be made between environment and the actual software.

Reply:
Will do it myself

Reply:
Indeed, the both should be dropped if we consider the 3 items.